acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/jsonrpc/Clique.java

```diff
@@ -98,6 +99,10 @@ private int cliqueBlockPeriod(final PantheonNode node) {
     return Arrays.stream(validators).map(PantheonNode::getAddress).sorted().toArray(Address[]::new);
   }
 
+  public Condition aBlockIsCreatedByProposer(final PantheonNode proposer) {
```

Comment:
Consistency; drop the `a` prefix == `blockIsCreatedByProposer`

Reply:
done